CHANGELOG-PENDING.md

```diff
@@ -7,7 +7,7 @@ Month, DD, YYYY
 ### BREAKING CHANGES
 
 - [go package] (Link to PR) Description @username
-[feat(cmd): give a birth to cel-shed and p2p key utilities #281](https://github.com/celestiaorg/celestia-node/pull/281) [@Wondertan](https://github.com/Wondertan)
+- [feat(cmd): give a birth to cel-shed and p2p key utilities #281](https://github.com/celestiaorg/celestia-node/pull/281) [@Wondertan](https://github.com/Wondertan)
```

Comment:
I guess this got included somehow on accident?

Reply:
I guess he just added `-` to make it a bullet point.